tls/s2n_security_policies.c

```diff
@@ -40,6 +40,19 @@ const struct s2n_security_policy security_policy_20190801 = {
     .ecc_preferences = &s2n_ecc_preferences_20200310,
 };
 
+const struct s2n_security_policy security_policy_20190802 = {
+    .minimum_protocol_version = S2N_TLS10,
+    .cipher_preferences = &cipher_preferences_20190801,
+    .kem_preferences = &kem_preferences_null,
+    /* The discrepancy in the date exists because the signature preferences
+     * were named when cipher preferences and signature preferences were
+     * tracked separately, and we chose to keep the cipher preference
+     * name because customers use it.
+     */
```

Comment:
The discrepancy in the date is present because we are using `s2n_signature_preferences_20200207` within `security_policy_20190802`.  The comment was to clarify this part. 

```
const struct s2n_security_policy security_policy_20190802 = {
    .minimum_protocol_version = S2N_TLS10,
    .cipher_preferences = &cipher_preferences_20190801,
    .kem_preferences = &kem_preferences_null,
    .signature_preferences = &s2n_signature_preferences_20200207,
    .ecc_preferences = &s2n_ecc_preferences_20140601,
};
```